hardhat.config.js

```diff
@@ -50,7 +50,7 @@ module.exports = {
     },
     goerli: {
       url: `https://goerli.infura.io/v3/${process.env.INFURA_KEY}`,
-      accounts: [process.env.GOERLI_PK],
+      accounts: process.env.GOERLI_PK,
```

Comment:
Why was this changed here (also why in other networks no)?

Reply:
Looks like the change breaks CI...

Reply:
it does fixing it